quadruped:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  # Position Controllers ---------------------------------------
  LF_HAA_position_controller:
    type: position_controllers/JointPositionController
    joint: LF_HAA
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  LF_HFE_position_controller:
    type: position_controllers/JointPositionController
    joint: LF_HFE
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  LF_KFE_position_controller:
    type: position_controllers/JointPositionController
    joint: LF_KFE
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  LH_HAA_position_controller:
    type: position_controllers/JointPositionController
    joint: LH_HAA
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  LH_HFE_position_controller:
    type: position_controllers/JointPositionController
    joint: LH_HFE
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  LH_KFE_position_controller:
    type: position_controllers/JointPositionController
    joint: LH_KFE
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  RF_HAA_position_controller:
    type: position_controllers/JointPositionController
    joint: RF_HAA
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  RF_HFE_position_controller:
    type: position_controllers/JointPositionController
    joint: RF_HFE
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  RF_KFE_position_controller:
    type: position_controllers/JointPositionController
    joint: RF_KFE
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  RH_HAA_position_controller:
    type: position_controllers/JointPositionController
    joint: RH_HAA
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  RH_HFE_position_controller:
    type: position_controllers/JointPositionController
    joint: RH_HFE
    #pid: {p: 100.0, i: 0.01, d: 10.0}
  RH_KFE_position_controller:
    type: position_controllers/JointPositionController
    joint: RH_KFE
    #pid: {p: 100.0, i: 0.01, d: 10.0}

gazebo_ros_control/pid_gains:
  LF_HAA: {p: 300.0, i: 0.01, d: 10.0}
  LH_HAA: {p: 300.0, i: 0.01, d: 10.0}
  RF_HAA: {p: 300.0, i: 0.01, d: 10.0}
  RH_HAA: {p: 300.0, i: 0.01, d: 10.0}
  LF_HFE: {p: 600.0, i: 0.01, d: 10.0}
  LH_HFE: {p: 600.0, i: 0.01, d: 10.0}
  RF_HFE: {p: 600.0, i: 0.01, d: 10.0}
  RH_HFE: {p: 600.0, i: 0.01, d: 10.0}
  LF_KFE: {p: 1000.0, i: 0.01, d: 10.0}
  LH_KFE: {p: 1000.0, i: 0.01, d: 10.0}
  RF_KFE: {p: 1000.0, i: 0.01, d: 10.0}
  RH_KFE: {p: 1000.0, i: 0.01, d: 10.0}
